### ISSUE TITLE
add/removeAttribute methods

### ISSUE THREAD

**Issue by mattgi** [balderdashy/sails-mongo#171]
Isn't it prudent to include these two methods in the Adapter to ensure the API is consistent across all adapters?

Whilst schema-less, a 'removeAttribute' method could remove an attribute from each document in a collection and an 'add' could add a default value if provided. This would allow data migrations to occur using existing libraries such as sails-migrations and ensure consistency across all major adapters.

(I know it may be an intensive task to update all documents within a collection, but its usage would be only out-of-process, so IMO it isn't really an issue).

**Comment by ArtworkAD** [comment]
:+1: this is crucial. But actually I am used to it to use native mongo for 50% of its features in sails.

**Comment by sailsbot** [comment]
Thanks for posting, @mattgi.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
